api/HTMLMediaElement.json

```diff
@@ -731,9 +732,16 @@
                 "alternative_name": "crossorigin"
               }
             ],
-            "firefox_android": {
-              "version_added": true
-            },
+            "firefox_android": [
+              {
+                "version_added": "22"
+              },
+              {
+                "version_added": "14",
+                "version_removed": "22",
+                "alternative_name": "crossorigin"
```

Comment:
Can you add this miscapitalized variant to the collector so that we can confirm the data as well eventually?

Reply:
Reopened, but doing this doesn't block this PR.

Reply:
Tracking alt. names (and prefixes) in https://github.com/foolip/mdn-bcd-collector/issues/333, so we can verify its data!